pkg/ptrace/stdwrapper_amd64.go

```diff
@@ -12,11 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+//go:build cgo
 
-package main
+package ptrace
 
-import "fmt"
+import (
+	"syscall"
+)
 
-func main() {
-	fmt.Println("linux arm64 is not supported")
```

Comment:
Can we merge this into ptrace_linux_xxx.go?

Reply:
fixed.